### ISSUE TITLE
[Feature Request] Show popup on hover over highlighted region

### ISSUE THREAD

**Issue by asaprahul** [caesarHQ/textSQL#49]
Currently, users have no easy way of getting additional data about a highlighted region when they hover over it. 

Mapbox supports showing a popup on hover ([link here](https://docs.mapbox.com/mapbox-gl-js/example/popup-on-hover/)):

<img width="350" alt="Screenshot 2023-03-11 at 1 29 40 AM" src="https://user-images.githubusercontent.com/10172332/224476596-048a77b8-616f-4a20-8b2a-a71ebbe854ef.png">

Expected behavior of this feature is when a user hovers over a highlighted region, we show them the corresponding data points (row) for that region from the rendered table

**Comment by blyncnov** [comment]
Can i work on this issue ?

**Comment by asaprahul** [comment]
@blyncnov yes! feel free to create a pull request. Let me know if you have any questions, we also have a discord for all contributors

**Comment by blyncnov** [comment]
alright @asaprahul  ... I joined the discord channel already

**Comment by blyncnov** [comment]
I have Configure the backend config.py and ran necessary commands . 

A development server is running, but there still seems to be some issue with it 

![20230312_115345.jpg](https://user-images.githubusercontent.com/66668114/224540017-5b0794ba-88ac-4a73-bb63-f266049c34e4.jpg)

![20230312_115459.jpg](https://user-images.githubusercontent.com/66668114/224540058-ab8a3172-70a8-4176-b525-3b3cd6305771.jpg)

**Comment by blyncnov** [comment]
I fixed that already 👌
